eventuals/builder.h

```diff
@@ -18,6 +18,14 @@ class Field;
 template <typename Value_>
 class Field<Value_, false> {
  public:
+  Field() = default;
+
+  Field(const Field&) = default;
```

Comment:
Why are some copy / move operations `noexcept` and others are not? I'd expect `noexcept` use to be consistent here: please document.

Glancing at code snippets below, I'd expect us to use `noexcept` everywhere we can.

Reply:
I don't think we have `noexcept` on copy operations (unless I made a typo?).

https://clang.llvm.org/extra/clang-tidy/checks/performance/noexcept-move-constructor.html

Reply:
> I don't think we have `noexcept` on copy operations (unless I made a typo?).

Why don't we have `noexcept` on our copy operations? Can our copy operations throw exceptions in a way that our move operation can't?

If not, it seems like our copy operations should also be `noexcept`.